stable/debian/Dockerfile

```diff
@@ -26,5 +27,5 @@ WORKDIR /etc/varnish
 COPY docker-varnish-entrypoint /usr/local/bin/
 ENTRYPOINT ["docker-varnish-entrypoint"]
 
-EXPOSE 80
-CMD ["varnishd", "-F", "-f", "/etc/varnish/default.vcl"]
+EXPOSE 80 8443
+CMD varnishd -F -f /etc/varnish/default.vcl -a http=:80,HTTP -a proxy=:8443,PROXY -s malloc,$SIZE
```

Comment:
Hi @gquintard how is it now possible to overwrite this values. We need to start it on port 8080 and put that value in the args before

Reply:
Hi,

You have multiple choices:
- have `docker` reroute the port with `-p` when you use `docker run`
- just add a `-a :8080` at the end of your `docker run` line
- provide the full `varnishd ...` line at the end of the `docker run` line

Reply:
Thanks for giving me some hints. I will a have a look into it. This change introduced an incident today, Pods couldn't start with the same image. (I was unaware that images can change even without updating the tag) but lesson learned :D

Reply:
Sorry about that, we are still figure out the proper interface. Feedback is definitely appreciated.

Which tag did you use?

Reply:
We used 6.0.6. I will think about feedback later, thank you. I switched down to 6.0.5 because that image was not touched with the changes.

Reply:
Thanks, so it looks like I am to blame, the `populate.sh` script just erase the old `6.0.6` definition, this is what screwed you over. Sorry about that, we'll try and be more precise in the future

Reply:
Is there a way to make it fall back to the previous behaviour?  I've (very gratefully!) been using this image in a Kubernetes cluster as part of a pod that's made up of an HAProxy container for SSL termination, forwarding to a Varnish container for caching, forwarding to a backend container, plus various other containers for log aggregation, monitoring, etc. Pulling in the `6.4.0-1` tag has caused two issues:
1. I get a port clash on port 8443: My HAProxy container fails to start because it listens on 8443 and Varnish has already snaffled the port by the time it starts
1. All the plaintext communication, i.e. from HAProxy on, is currently over Unix Domain Sockets.  This gives me greater control over which containers (and hence what 3rd party code) gets access to the plaintext HTTP traffic, because I can make the sockets only available to the appropriate containers.  I feel a bit uncomfortable having ports 80 and 8443 opened on 0.0.0.0

Reply:
Hi,

This is just a matter of overriding the command and just using:

```
docker run varnish varnishd -F -f /etc/varnish/default.vcl
```

isn't it?

Reply:
Yes, you're right.

I suppose the point I was rather awkwardly trying to make is that opening up ports is a breaking change, most likely to impact users who are passing -a arguments to the entrypoint already.

On that basis, I was wondering if other people might be saved from some head scratching (or just inadvertent port opening) by attempting to maintain the behaviour prior to this merge when entrypoint arguments are supplied.

On the other hand, maybe I should be more diligent in reading release notes when taking a new version...

Reply:
I totally get your point, but now the problem is that reverting that behavior would be another breaking change, so I'm not really keen on doing that.
Can I offer you to send an email to varnish-misc so we can discuss that with a broader audience and decide what users think is best?